client/js/lounge.js

```diff
@@ -628,12 +628,23 @@ $(function() {
 		return false;
 	});
 
+	// Used when resetting original height after deleting a character
+	var initialInputHeight;
+
 	var input = $("#input")
 		.history()
 		.on("input keyup", function() {
 			var style = window.getComputedStyle(this);
-			this.style.height = "0px";
-			this.offsetHeight; // force reflow
+
+			if (!initialInputHeight) {
+				// When first resizing the textarea, store its initial height for later
+				initialInputHeight = this.style.height;
```

Comment:
Actually, there is a height, not a min-height at the moment in https://github.com/thelounge/lounge/blob/ebab910e0013b15ff0bcc754b2003fb40e7d2298/client/css/style.css#L1262-L1275, so I need to add it. Will do.